core/src/sumeragi/network_topology.rs

```diff
@@ -277,7 +251,7 @@ impl Topology {
 
     /// Answers if the consensus stage is required with the current number of peers.
     pub fn is_consensus_required(&self) -> bool {
-        self.sorted_peers.len() > 1
+        self.min_votes_for_commit() > 1
```

Comment:
This also implies `self.max_faults() > 0` or `self.sorted_peers.len() > 3`.
Conversely, if there are 2 or 3 peers the leader commits at its own discretion as f is 0, but needs to broadcast the committed block

Reply:
what you're saying is there is no point in consensus if the system is bellow byzantine fault tolerance limit

Reply:
What does "BFT limit" mean?
The system just has f tolerance according to its topology length. f is neither below nor above f

Reply:
ok, no point in reaching consensus for <= 3